src/Accelerators/NNPA/Conversion/ONNXToZHigh/ONNXLegalityCheck.hpp

```diff
@@ -43,5 +44,11 @@ mlir::StringRef getStrPaddingType(OP op);
 /// Check if input, output, kernel, strides, and paddingYype for each axis meet
 /// parameter restrictions for maxpool and avgpool.
 /// See "MaxPool2D/AvgPool2D Parameter Restrictions" in "zDNN API Reference"
-bool meetPoolParamRestrictions(int64_t inputShape, int64_t kernelShape,
-    int64_t strides, int64_t outputShape, mlir::StringRef paddingType);
+bool meetPoolParamRestrictions(mlir::Operation *op, int64_t inputShape,
+    int64_t kernelShape, int64_t strides, int64_t outputShape,
+    mlir::StringRef paddingType);
+
+bool emitWarningMessageNNPAUnsupported(
+    mlir::Operation *op, const std::string &message);
+
+bool emitWarningMessageInCompatibility(mlir::Operation *op);
```

Comment:
Would you mind if we change these function names to make it consistent with functions for other `--op-report` options? We have `onnxToKrnlSimdReport` and `onnxToKrnlParallelReport`,  so how about `onnxToZHighUnsupportedReport`? or feel free to choose another name.

Reply:
Done. Thanks!